### PR TITLE
Fixes #1420 : .yaydoc.yml added to support yaydoc

### LINKED ISSUE #1420: Add .yaydoc.yml to integrate yaydoc
[Yaydoc](http://github.com/sch00lb0y/yaydoc) is a documentation generation project maintained by FOSSASIA. Implement yaydoc in `open-event` projects to generate documentation.

In order to generate documentation for sub project. sub project also needs `.yaydoc.yml`.
so add the `.yaydoc.yml` configuration to the `open-event-webapp`

### PR DESCRIPTION
**Pull request by sch00lb0y**
Fixes #1420 

Changes: .yaydoc.yml is added for documentation generation and also fixed markdown for proper generation of documentation

Screenshots for the change:

### CODE REVIEW

**Comment by sch00lb0y** [comment]
@pri22296 @imujjwal96 please check this

**Comment by pri22296** [comment]
@sch00lb0y Can you share a link to the generated docs

**Comment by sch00lb0y** [comment]
@pri22296 http://yaydoc.herokuapp.com/preview/rbalajis25@gmail.com/0a717245-ed2c-41af-9231-0ea80d4c4516_preview/

**Comment by imujjwal96** [comment]
@sch00lb0y  Check the global toctree, you'll have to make changes

**Comment by sch00lb0y** [comment]
@imujjwal96 are you asking me to add manual rst ?

**Comment by imujjwal96** [comment]
image

Here automatic deploy actually describe `INSTALLATION_HEROKU`. Change INSTALLATION_HEROKU.MD  by changing # headings to ## and creating a # heading
`# Heroku deployment docs` or similar

**Comment by sch00lb0y** [comment]
@imujjwal96 @pri22296 I have updated http://yaydoc.herokuapp.com/preview/rbalajis25@gmail.com/5ce7f26b-ee15-4b9f-af83-ebfc140fc28e_preview/

File: .yaydoc.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+metadata:
+  author: FOSSASIA
+  projectname: open-event-webapp
```
Comment (mariobehling) [inline .yaydoc.yml]: Correct name is "Open Event Webapp"

**Comment by sch00lb0y** [comment]
@mariobehling right now I'm fixing all the markdown issues in the sub project. The CNAME will be defined in the main project and it'll get deployed based on the main project configuration.

File: .yaydoc.yml (hunk 1 above)
Comment (sch00lb0y) [inline .yaydoc.yml]: Updated. Thanks

**Comment by mariobehling** [comment]
@sch00lb0y This repo already has a gh-pages branch. Please make sure, that you don't use that.

**Comment by sch00lb0y** [comment]
@mariobehling sure, documentation won't get deployment to the gh-pages

**Pull request merged**